### DISCUSSION TITLE
Why is imageSizes being applied even though I haven't set the 'sizes' prop?

### DISCUSSION THREAD

**Discussion by hwajj** [vercel/next.js#55133] [Help]
### Summary

According to the documentation, it says, 
'The reason there are two separate lists is that imageSizes is only used for images which provide a sizes prop, which indicates that the image is less than the full width of the screen.' 

Since I haven't configured deviceSizes separately, the smallest size for deviceSizes is 640." 
Also, I didn't set **sizes** prop so I didn't expect it to apply to imageSizes. 
However, when I loaded an image with dpr=1 and width=100 on my computer, 
I was able to download an image with w=128. 
According to the documentation, shouldn't it download an image with w=640? 
I haven't set any **sizes**,  so why am I able to load a w=128 image?

I would really appreciate your help and insights on this matter. Thank you in advance!

**Comment by philwolstenholme** [comment]
I came here to post the same question. If I ask Next for an image of `width="120" height="120"` with no `sizes` prop I'd expect it to generate an image sized exactly `120x120`, not `128x128` or whatever else value from `imageSizes`.

Is this a bug I wonder?

```
❯ npx next info

Operating System:
  Platform: darwin
  Arch: x64
  Version: Darwin Kernel Version 22.6.0: Fri Sep 15 13:41:28 PDT 2023; root:xnu-8796.141.3.700.8~1/RELEASE_ARM64_T6000
Binaries:
  Node: 18.18.0
  npm: 10.2.0
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 13.5.6
  eslint-config-next: 13.5.6
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.2.2
Next.js Config:
  output: N/A
```
